### DISCUSSION TITLE
Grayscale image conversion to RGB using R programming

### DISCUSSION THREAD

**Discussion by Mr-Jay13** [community/community#133293] [Programming Help]
### Body

How to convert grayscale images to RGB in R programming? I'm making a CNN.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by AtlasKeeper** [comment]
`install.packages("imager")`
`library(imager)`
`gray_image <- load.image("path/to/your/grayscale_image.png")`
`rgb_image <- as.cimg(array(rep(gray_image, 3), dim = c(dim(gray_image), 3)))`
`save.image(rgb_image, "path/to/your/rgb_image.png")`
